DEVELOPING.md

```diff
@@ -2,9 +2,31 @@
 
 In addition to providing out-of-the-box forests for quantile regression and instrumental variables, grf provides a framework for creating forests tailored to new statistical tasks. Certain components around splitting and prediction can be swapped out, within the general infrastructure for growing and predicting on trees.
 
-### Working with the code
+## Contributing
 
-The core forest implementation is written in C++, with an R interface powered by Rcpp. We recommend using a full-powered C++ IDE such as CLion, Xcode, or Visual Studio when working with the core code. To build the R package from source, cd into `r-package` and run the `build_package.R` script. Code style consistency is checked with the [lintr](https://github.com/jimhester/lintr) package, and in RStudio you can make sure the code is consistent with the R standard by clicking _Addins->Style_.
+This repository follows the standard open source protocol and setup with git where there are an abundance of existing resources to get up to speed (see for example the
```

Comment:
I have some ideas for fleshing out this section, but we can discuss that in a follow-up PR.